packages/ButtonLink/ButtonLink.md

```diff
@@ -16,6 +10,82 @@ By default, Buttons will be displayed in the `primary` variant. Use primary Butt
 - Make sure the button text describes an action
 - Buttons should not be disabled
 
+### OLD ButtonLink variants
```

Comment:
Let's just include a link to the old/new variant section of the `Button` component.